doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -27,19 +50,22 @@ ifeval::["{context}"!="openshift-immutable"]
 Alternatively, if you use a custom external repository (for example, Nexus) for your services, you can use the same repository as a mirror repository.
 endif::[]
 +
+endif::offline_onprem[]
 . On the computer that has an outgoing connection to the public Internet, complete the following steps:
 .. Download the latest version of the http://release-engineering.github.io/offliner/[Offliner tool].
 .. Download the `{PRODUCT_FILE}-offliner.txt` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
 .. Enter the following command to download the required artifacts using the Offliner tool:
```

Comment:
Enter the following command to use the Offliner tool to download the required artifacts.